### DISCUSSION TITLE
Notifications are not sent to e-mail.

### DISCUSSION THREAD

**Discussion by akuleckij** [wazuh/wazuh#14257] [Q&A] [answered]
Installed and configured Postfix. The test message passes normally:
echo "Test mail from postfix" | mail -s "Test Postfix" -r "хххх@example.com" хххх@example.com

Configured wazuh:
<global>
    <jsonout_output>yes</jsonout_output>
    <alerts_log>yes</alerts_log>
    <logall>no</logall>
    <logall_json>no</logall_json>
    <email_notification>yes</email_notification>
    <smtp_server>localhost</smtp_server>
    <email_from>хххх@example.com</email_from>
    <email_to>хххх@example.com</email_to>
    <email_maxperhour>12</email_maxperhour>
    <email_log_source>alerts.log</email_log_source>
    <agents_disconnection_time>10m</agents_disconnection_time>
    <agents_disconnection_alert_time>0</agents_disconnection_alert_time>
  </global>
  
 <reports>
      <level>10</level>
      <title>Daily report: Alerts with level higher than 10</title>
      <email_to>хххх@example.com</email_to>
  </reports>
  
  <alerts>
    <log_alert_level>3</log_alert_level>
    <email_alert_level>6</email_alert_level>
  </alerts>
  
But notifications do not come to the mail.
Help me please.

**Comment by mdiego92** [comment, marked as answer]
Hello Akuleckij,

We're sending you the documentation for the SMTP configuration:
**Article for reference**: https://documentation.wazuh.com/current/user-manual/manager/manual-email-report/smtp-authentication.html

What's the email service you're trying to use to receive these emails? Remember that there are some email servers that require some specific configuration to receive.

Please let us know if you have any questions or issues.

Thank you very much.

**Reply by akuleckij** [reply]
> Здравствуйте, Акулецкий.
> 
> Мы отправляем вам документацию по настройке SMTP: **Статья для справки** : https://documentation.wazuh.com/current/user-manual/manager/manual-email-report/smtp-authentication.html
> 
> Какую службу электронной почты вы пытаетесь использовать для получения этих писем? Помните, что есть некоторые почтовые серверы, для получения которых требуется определенная конфигурация.
> 
> Пожалуйста, дайте нам знать, если у вас есть какие-либо вопросы или проблемы.
> 
> Большое спасибо.

I use smtp.mail.ru and the test message normally goes away, messages from wazuh do not go away.
I mean, postfix sends test messages, but wash does not send messages.

**Reply by mdiego92** [reply]
Hello Akuleckij,

Were you able to configure your `/var/ossec/etc/ossec.conf` file with your smtp.mail ru settings?

E.g.:

```
<global>
  <email_notification>yes</email_notification>
  <smtp_server>localhost</smtp_server>
  <email_from>USERNAME@gmail.com</email_from>
  <email_to>you@example.com</email_to>
</global>
```

Also, it is important to remark that you will receive the alerts depending on your configured alert level:

`<email_alert_level>12</email_alert_level>`

As you can see, it is set to 12 by default.


Please let us know if you have any remaining questions.
Thank you very much.

**Reply by akuleckij** [reply]
Good afternoon, yes, I made the settings, but there was no notification. How do I check if wazuh tried to send anything?

**Reply by mdiego92** [reply]
Hello Akuleckij,

You told us that when testing postfix configuration you're receiving emails, is that correct?

Could you please verify if any events met the criteria configured for sending out an email?

`cat /var/ossec/logs/alerts/alerts.json | grep '"mail":true'`

Also, you could confirm if there are any error messages from the Wazuh mail daemon and within the mail log respectively:

`cat /var/ossec/logs/ossec.log | grep mail
`
`cat /var/log/maillog | grep error`

Thank you very much.
